src/components/DatetimePicker/DatetimePicker.vue

```diff
@@ -49,6 +51,28 @@ export default {
 		Datepicker
 	},
 
-	inheritAttrs: false
+	inheritAttrs: false,
+
+	computed: {
+		firstDay() {
+			return window.firstDay
+				? window.firstDay
+				: 0 // sunday as default
```

Comment:
default should be 7

Reply:
as European, yes :)
Not for the rest of the world?

Reply:
@jancborchardt ^

Reply:
0 is not working in fact, it returns an error; also, based on documentation:


Prop | Description | Type | Default
-- | -- | -- | --
first-day-of-week | set the first day of week | 1 - 7 | 7
-- | -- | -- | --

Reply:
oh damn

Reply:
> as European, yes :)
> Not for the rest of the world?

Monday as first day of the week is ISO standard. So while yeah, US, Canada, Australia and others don’t do that, we just need to pick something. :) Australia and Canada also adopted the metric system some time ago, so there’s hope. ;)